docs/guide/front-end-helpers.md

```diff
@@ -0,0 +1,319 @@
+# Front end helper methods
+
+Apostrophe provides a small library of front-end utility methods to support implementing client-side Javascript. These can be useful in widget players, for example. [General utility methods](#general-utility-methods) area available on `apos.util` and [HTTP request methods](#http-request-methods) are available on `apos.http`.
+
+## General utility methods
+
+These are all available in the browser on `apos.util`, e.g., `apos.util.addClass(el, 'is-active')`. They include wrappers for common browser APIs, classic DOM traversal methods, and Apostrophe-specific utilities.
```

Comment:
They are compatible with all modern browsers, and also with Internet Explorer 11.